nsv13/code/modules/overmap/radar.dm

```diff
@@ -1,8 +1,22 @@
+/obj/item/circuitboard/computer/ship/dradis
```

Comment:
if you're adding DRADIS circuit boards, can you add a way to replace the mining ship DRADIS console?

Reply:
How is the mining ship DRADIS different?